jupyterlab_latex/config.py

```diff
@@ -21,3 +21,7 @@ class LatexConfig(Configurable):
         'Can be "restricted", for restricted shell escapes, '+\
         '"allow", to allow all shell escapes, or "disallow", '+\
         'to disallow all shell escapes')
+    run_times = Integer(default_value=1, config=True,
+        help='How many times to compile the ".tex" files.')
+    cleanup_setting = Bool(default_value=True, config=True,
```

Comment:
Since it's already a setting, maybe just name it `cleanup`?

Reply:
will do